test/quantization/core/test_quantized_op.py

```diff
@@ -6233,6 +6233,30 @@ def test_conv_reorder_issue_onednn(self):
             # The following should pass when input shape is changed
             torch.ops.quantized.conv2d(qx, w_packed, output_scale=1.0, output_zero_point=0)
 
+    def test_conv_transpose_reorder_issue_onednn(self):
+        if 'onednn' not in supported_qengines:
+            return
```

Comment:
I think you can use: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L3992

Reply:
Good suggestions. Changed it.